Where2Play/Models/EventSummary.cs

```diff
@@ -1,22 +1,19 @@
-using System;
+namespace Where2Play.Models;
 
-namespace Where2Play.Models
+/// <summary>
+/// Combines artist and concert information from MusicBrainz and Setlist.fm.
+public class EventSummary
 {
-    /// <summary>
-    /// Combines artist and concert information from MusicBrainz and Setlist.fm.
-    public class EventSummary
-    {
-        // Artist info (MusicBrainz)
-        public string ArtistName { get; set; }
-        public string Genre { get; set; }
-        public string Country { get; set; }
+    // Artist info (MusicBrainz)
+    public string ArtistName { get; set; }
+    public string Genre { get; set; }
+    public string Country { get; set; }
```

Comment:
These should be indented, as they are within a {}.
Each time you have a {}, indent the contents by one tab for readability.